<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸考勤系统</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cameraView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openCameraBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QListView" name="userListView">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>30</y>
      <width>181</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>-10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>已签到人员：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="userNumLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>440</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0人/30人</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>291</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>手动录入</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="recordBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>录入</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>291</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>当前信息</string>
    </property>
    <widget class="QLabel" name="curUserImg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>256</width>
       <height>192</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 87, 83);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="curUserName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="curUserNumber">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="curTimeCost">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>271</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>耗时：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1133</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openDatabaseMenu"/>
    <addaction name="closeDatabaseMenu"/>
    <addaction name="exportDatabaseMenu"/>
    <addaction name="exitMenu"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>人脸管理</string>
    </property>
    <addaction name="insertFaceTemplateMenu"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>组织管理</string>
    </property>
    <addaction name="newOrganizationMenu"/>
    <addaction name="editOrganizationMenu"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openDatabaseMenu">
   <property name="text">
    <string>打开数据库文件</string>
   </property>
  </action>
  <action name="closeDatabaseMenu">
   <property name="text">
    <string>关闭数据库文件</string>
   </property>
  </action>
  <action name="saveDatabaseMenu">
   <property name="text">
    <string>保存数据库文件</string>
   </property>
  </action>
  <action name="exportDatabaseMenu">
   <property name="text">
    <string>导出数据库文件</string>
   </property>
  </action>
  <action name="showFaceTemplateMenu">
   <property name="text">
    <string>人脸模板查看</string>
   </property>
  </action>
  <action name="insertFaceTemplateMenu">
   <property name="text">
    <string>人脸模板录入</string>
   </property>
  </action>
  <action name="editFaceTemplateMenu">
   <property name="text">
    <string>人脸模板编辑</string>
   </property>
  </action>
  <action name="exitMenu">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="documentMenu">
   <property name="text">
    <string>使用文档</string>
   </property>
  </action>
  <action name="aboutMenu">
   <property name="text">
    <string>关于本系统</string>
   </property>
  </action>
  <action name="newOrganizationMenu">
   <property name="text">
    <string>新建组织</string>
   </property>
  </action>
  <action name="showOrganizationMenu">
   <property name="text">
    <string>组织一览</string>
   </property>
  </action>
  <action name="editOrganizationMenu">
   <property name="text">
    <string>编辑组织</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
